src/App.js

```diff
@@ -1,25 +1,78 @@
-import logo from './logo.svg';
 import './App.css';
+import Form from './Form';
+import DisplayLegoSets from './DisplayLegoSets';
+import LegoHouse from './LegoHouse';
+import legoHeads from './assets/legoHeads.png';
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+
+  const [legoSets, setLegoSets] = useState([]);
+
+  const [userInputValues, setUserInputValues] = useState([]);
+
+
+  useEffect(() => {
+    // make an axios call to the Rebrickable Api to get back an array of lego sets
+
+    if (userInputValues  && userInputValues.length !== 0 && userInputValues.year !== "") {
+      axios({
+        baseURL: 'https://rebrickable.com',
+        url: '/api/v3/lego/sets/',
+        method: 'GET',
```

Comment:
axis method is by default GET, if you want to shorten your code a tiny bit you can remove this line